graphql/schemas/Ecosystem/templates.graphql

```diff
@@ -0,0 +1,59 @@
+type Templates {
+    id: ID!
+    users_id: Int!
+    companies_id: Int!
+    apps_id: Int!
+    name: String!
+    parent_template_id: Int!
+    template: String!
+    template_variables: [TemplatesVariables!]@hasMany(relation: "templateVariables")
+    created_at: String
+    updated_at: String
+    is_deleted: Int
+}
+
+input TemplatesInput {
+    name: String!
+    parent_template_id: Int!
+    template_variables: [TemplateVariableInput!]!
+    template: String!
+}
+
+type TemplatesVariables {
+    name: String
+    value: String
+    template_id: Int
+}
+
+
+input TemplateVariableInput {
+    key: String!
+    value: String!
+}
+
+extend type Mutation @guard {
+    createOrUpdateTemplate(input: TemplatesInput!): Templates!
```

Comment:
@rwhite27 devide create and update even if we use the same method just for verbose